pyo3-macros-backend/src/pyclass.rs

```diff
@@ -338,11 +338,11 @@ struct PyClassEnum<'a> {
     // The underlying #[repr] of the enum, used to implement __int__ and __richcmp__.
     // This matters when the underlying representation may not fit in `isize`.
     repr_type: syn::Ident,
-    variants: Vec<PyClassEnumVariant<'a>>,
+    variants: Vec<(PyClassEnumVariant<'a>, EnumVariantPyO3Options)>,
```

Comment:
Can `EnumVariantPyO3Options` be added to `PyClassEnumVariant`? Would be a bit nicer to have a `.options` field imo rather than a vec of tuples

Reply:
I implemented it this way to match what is done with struct fields. This would make it easier if in the future we want to replace `PyClassEnumVariant` with `syn::Field` as with struct fields. I think this would mainly be needed when adding support for non-C-like enums.